src/views/BottomTabBar.js

```diff
@@ -208,6 +209,8 @@ class TabBarBottom extends React.Component<Props> {
     return (
       <SafeAreaView style={tabBarStyle} forceInset={safeAreaInset}>
         {routes.map((route, index) => {
+          route.index = index;
+          route.routesLength = routesLength;
```

Comment:
We shouldn't be mutating `route`. Also, don't think these properties should be on the route as the public API.